### ISSUE TITLE
Content attribute type array instead of string in 03-editable block examples

### ISSUE THREAD

**Issue by tloureiro** [WordPress/gutenberg-examples#220]
Why the type of content attribute of the 03-editable (and 03-editable-jsx) examples is array and not string? I know they work correctly but wouldn't string be a more appropriate type?

Thanks!

**Comment by ryanwelcher** [comment]
It looks like the [JSX version is using `string`](https://github.com/WordPress/gutenberg-examples/blob/trunk/blocks-jsx/03-editable-esnext/block.json#L11) and the [non-JSX version is using `array`](https://github.com/WordPress/gutenberg-examples/blob/trunk/blocks-non-jsx/03-editable/block.js#L20).

We should probably have them both set to `string`. Interested in putting together a pull request?

**Comment by tloureiro** [comment]
Yes, here it is (sorry for the delay) https://github.com/WordPress/gutenberg-examples/pull/228
